core/src/main/java/com/linecorp/armeria/common/logging/RequestLogBuilder.java

```diff
@@ -269,6 +276,12 @@ void startRequest(Channel channel, SessionProtocol sessionProtocol, @Nullable SS
      */
     void increaseResponseLength(long deltaBytes);
```

Comment:
ditto. There are two other usages in `HttpServerHanlder` and `HttpResponseSubscriber`.